fitbenchmarking/results_processing/plots.py

```diff
@@ -514,7 +522,9 @@ def plot_residuals(cls, categories, title, options, figures_dir) -> str:
             if row_ind == 1:
                 fig.update_traces(row=row_ind)
 
-        fig.update_layout(title=title + ": residuals")
+        fig.update_layout(
+            title=title + ": residuals", legend=cls._legend_style
+        )
```

Comment:
```suggestion
        fig.update_layout(title=title + ": residuals")
```

Is it possible to revert this change ? Because in this case having the legend on the right side results in a bigger plot. The issue with the extended names will not occur for the residual plots. Because the minimizer names won't be appended with the software and jacobian type (as in the other plots).

<img width="1000" alt="Screenshot 2025-05-09 at 15 30 41" src="https://github.com/user-attachments/assets/a8253f84-4572-4aaa-a073-5ce900dc7924" />


<img width="1000" alt="Screenshot 2025-05-09 at 15 34 41" src="https://github.com/user-attachments/assets/776ff01a-60eb-4cda-bd3a-ca12636a0371" />

<img width="1000" alt="Screenshot 2025-05-09 at 15 23 43" src="https://github.com/user-attachments/assets/10e95c69-694f-45ab-affe-a1d3f6334d50" />

Reply:
Makes sense, thanks for checking! Commited this now